lib/rules/require-prop-type-constructor.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @fileoverview require prop type to be a constructor
+ * @author Michał Sajnóg
+ */
+'use strict'
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+const message = 'The "{{name}}" property should be a constructor.'
+
+const forbiddenTypes = [
+  'Literal',
+  'TemplateLiteral',
+  'BinaryExpression',
+  'UpdateExpression'
+]
+
+const isForbiddenType = nodeType => forbiddenTypes.indexOf(nodeType) > -1
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'require prop type to be a constructor',
+      category: undefined, // essential
+      url: 'https://github.com/vuejs/eslint-plugin-vue/blob/v5.0.0-beta.1/docs/rules/require-prop-type-constructor.md'
+    },
+    fixable: 'code',  // or "code" or "whitespace"
+    schema: []
+  },
+
+  create (context) {
+    const fix = node => fixer => {
+      if (node.type === 'Literal') {
+        return fixer.replaceText(node, node.value)
```

Comment:
It is need to check if the value type is string.
Hexadecimal literals such as `0xF` are converted to numeric literals like `15`.

Reply:
I didn't consider this case, as it seems extremely rare and nearly impossible to get unless you really want to break it.

Reply:
> extremely rare and nearly impossible to get unless you really want to break it.

I think that you are right. I do not think it is necessary to take time and fix it.